modules/mono/editor/GodotTools/GodotTools.IdeMessaging/IHandshake.cs

```diff
@@ -3,6 +3,6 @@ namespace GodotTools.IdeMessaging
     public interface IHandshake
     {
         string GetHandshakeLine(string identity);
-        bool IsValidPeerHandshake(string handshake, out string identity, ILogger logger);
+        bool IsValidPeerHandshake(string handshake, out string? identity, ILogger logger);
```

Comment:
```suggestion
        bool IsValidPeerHandshake(string handshake, [NotNullWhen(true)] out string? identity, ILogger logger);
```

Reply:
Surprisingly, this attribute isn't available in .NET Standard 2.0

Reply:
Well, that's not really surprising since these attributes didn't exist when .NET Standard 2.0 came out. But yeah I missed that this project is targeting `netstandard2.0`. We can, however, add the attribute ourselves since the compiler only cares about the shape.

See https://github.com/godotengine/godot/blob/0cce6eb1505f7dd9c0cece2a825b1509cf23f158/modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/CodeAnalysisAttributes.cs

Reply:
Huh! Well that makes matters more convenient